plugins/remotelog/plugin.go

```diff
@@ -0,0 +1,69 @@
+// remotelog is a plugin that enables log messages being sent via UDP to a central ELK stack for debugging.
+// It is disabled by default and when enabled, additionally, logger.disableEvents=false in config.json needs to be set.
+// The destination can be set via logger.remotelog.serverAddress.
+// All events according to logger.level in config.json are sent.
+package remotelog
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/iotaledger/goshimmer/packages/parameter"
+	"github.com/iotaledger/goshimmer/plugins/autopeering/local"
+	"github.com/iotaledger/hive.go/events"
+	"github.com/iotaledger/hive.go/logger"
+	"github.com/iotaledger/hive.go/node"
+)
+
+type LogMessage struct {
+	NodeId    string    `json:"nodeId"`
+	Level     string    `json:"level"`
+	Name      string    `json:"name"`
+	Msg       string    `json:"msg"`
+	Timestamp time.Time `json:"timestamp"`
+}
+
+const (
+	CFG_SERVER_ADDRESS = "logger.remotelog.serverAddress"
+	CFG_DISABLE_EVENTS = "logger.disableEvents"
+	PLUGIN_NAME        = "remotelog"
+)
+
+var (
+	PLUGIN = node.NewPlugin(PLUGIN_NAME, node.Disabled, configure)
+	conn   net.Conn
+	myID   string
+)
+
+func configure(plugin *node.Plugin) {
+	log := logger.NewLogger(PLUGIN_NAME)
+
+	if parameter.NodeConfig.GetBool(CFG_DISABLE_EVENTS) {
+		log.Warnf("%s in config.json needs to be false so that events can be captured!", CFG_DISABLE_EVENTS)
```

Comment:
Call Fatalf instead in order to let the node crash immediately instead of letting it run with this plugin not working properly.